frontend/src/views/camp/Material.vue

```diff
@@ -1,145 +1,41 @@
 <!--
-Admin screen of a camp: Displays MaterialLists and MaterialItems
+Show all material lists for a camp on mobile
 -->
 
 <template>
   <content-card :title="$tc('views.camp.material.title')" toolbar>
     <template #title-actions>
-      <v-btn small color="primary" class="ml-5" @click="downloadXlsx">
-        <v-icon left>mdi-microsoft-excel</v-icon>
-        {{ $tc('views.camp.material.downloadXlsx') }}
-      </v-btn>
+      <DialogMaterialListCreate :camp="camp()">
+        <template #activator="{ on }">
+          <ButtonAdd class="mr-n2" height="32" v-on="on">{{
+            $tc('global.button.create')
+          }}</ButtonAdd>
+        </template>
+      </DialogMaterialListCreate>
     </template>
-    <v-card-text>
-      <e-switch
```

Comment:
I'm not sure what the use case for this is. I think we should look at this in a separate issue, where we can tackle the confusion about period/activity material as well. For me this is the much bigger problem. Now you can just look at the individual lists which will alleviate the issue with loading.

If you think we should look at this in this issue, then I would suggest making the lastColumn sortable (based on type: `activity`/ `period`)

Reply:
> I'm not sure what the use case for this is.

Let's take a 2 week camp. This could easily include a few hundred activity material items (14 days * 5 activities * 5 material items = 350 items). Now I want to add/edit my period/camp material (a few items only). In order to so so, i need to
- Load all these other 350 material items
- Visually navigate around these 350 material items which I'm not interested in. This is made even harder because at the moment, all period material is sorted at the beginning, but the form to add new items is at the bottom.

I'm not saying the hide/show toggle is the best and only solution. But above user experience is not very nice.

I'm also fine if we tackle this in a separate issue. However, in this PR we're removing code that allows to show/hide activity material which we might add again if we decide show/hide actually is the best solution. So I wonder if it's not easier to keep this toggle in at the moment (even if it's visually not very nice), and remove it later once we have a better solution for it.

Reply:
What do you think about: 0780683

Reply:
@usu I added a column header filter:
Unfiltered: ![Material list with "Activity / Period" column header and a outlined filter icon](https://github.com/ecamp/ecamp3/assets/3001985/a9387a61-dba8-4e5f-a53d-aa0719ad1858)

Filtered: ![Material list with blue "Filter: Period" column header and a blue bold filter icon](https://github.com/ecamp/ecamp3/assets/3001985/20a0562f-951e-41e7-87c0-0bd1c2379c19)

Reply:
😃 The functionality is there. The UX I'm not sure. But I think this is easier to discuss face-to-face + maybe trying to get some user feedback on how they use or want to use the material feature.

Reply:
Okay, maybe it is really better to discuss this at the hackathon. I still don't get the importance and think this whole mental model period vs activity material isn't helping our users and has a bad UX.